src/main/kotlin/com/cognifide/gradle/aem/common/instance/Instance.kt

```diff
@@ -300,23 +300,26 @@ open class Instance(@Transient @get:JsonIgnore protected val aem: AemExtension)
             }.distinct()
 
             return instanceNames.mapNotNull { name ->
-                val props = allProps.filterKeys {
-                    Patterns.wildcard(it, "instance.$name.*")
-                }.entries.fold(mutableMapOf<String, String>()) { result, e ->
-                    val (key, value) = e
-                    val prop = key.substringAfter("instance.$name.")
-                    result.apply { put(prop, value as String) }
-                }
+                val defaultProps = prefixedProperties(allProps, "instance.default")
```

Comment:
this is a new feature. the common place to define properties / no need to repeat for each instance separately